src/mono/wasm/build/WasmApp.targets

```diff
@@ -65,6 +65,7 @@
       - $(RunAOTCompilationAfterBuild)      - Run AOT compilation even after Build. By default, it is run only for publish.
                                               Defaults to false.
       - $(WasmAotProfilePath)               - Path to an AOT profile file.
+      - $(WasmExceptionHandling)            - Enable support for the WASM Exception Handling feature.
```

Comment:
This should be renamed to `WasmEnableExceptionHandling`.

Reply:
that sounds like there wouldn't be exception handling support if it isn't true though

Reply:
WasmEnableWasmEHFeature? WasmEnableNativeExceptionHandlingFeature?